src/Models/Stores.php

```
@@ -7,9 +7,9 @@
 use OpenFGA\Schema\CollectionSchemaInterface;
 
 /**
- * @extends AbstractIndexedCollection<Store>
+ * @extends IndexedCollection<Store>
  */
-final class Stores extends AbstractIndexedCollection implements StoresInterface
+final class Stores extends IndexedCollection implements StoresInterface
```

PHPStan (2.1.15)

Class OpenFGA\Models\IndexedCollection specifies template type T of interface OpenFGA\Models\IndexedCollectionInterface as OpenFGA\Models\Store but it's already specified as OpenFGA\Models\StoreInterface.


(generics.interfaceConflict)